which chromium version  do you use?

dzhj11 opened the pull request:
which chromium version do you use ? Is it a stable version

davisford committed:
update with rebuilt Chromium for Android/ARM July 2013

davisford committed:
Update README.md

davisford committed:
Update README.md

davisford committed:
Update README.md

pwnall commented:
@dzhj11 Sorry, I can't pull this version. It has the same issue as the build I'm putting together -- it crashes when I try to input in a text field.

```
07-18 11:35:47.337: W/ResourceType(30172): No package identifier when getting value for resource number 0x00000000
07-18 11:35:47.337: W/System.err(30172): android.content.res.Resources$NotFoundException: Resource ID #0x0
07-18 11:35:47.337: W/System.err(30172):    at android.content.res.Resources.getValue(Resources.java:1014)
07-18 11:35:47.337: W/System.err(30172):    at android.content.res.Resources.loadXmlResourceParser(Resources.java:2139)
07-18 11:35:47.337: W/System.err(30172):    at android.content.res.Resources.getLayout(Resources.java:853)
07-18 11:35:47.337: W/System.err(30172):    at android.view.LayoutInflater.inflate(LayoutInflater.java:394)
07-18 11:35:47.337: W/System.err(30172):    at android.view.LayoutInflater.inflate(LayoutInflater.java:352)
07-18 11:35:47.337: W/System.err(30172):    at org.chromium.ui.autofill.AutofillPopup.getDesiredWidth(AutofillPopup.java:177)
07-18 11:35:47.337: W/System.err(30172):    at org.chromium.ui.autofill.AutofillPopup.show(AutofillPopup.java:145)
07-18 11:35:47.337: W/System.err(30172):    at org.chromium.android_webview.AwAutofillManagerDelegate.showAutofillPopup(AwAutofillManagerDelegate.java:63)
07-18 11:35:47.337: W/System.err(30172):    at org.chromium.base.SystemMessageHandler.nativeDoRunLoopOnce(Native Method)
07-18 11:35:47.337: W/System.err(30172):    at org.chromium.base.SystemMessageHandler.handleMessage(SystemMessageHandler.java:27)
07-18 11:35:47.337: W/System.err(30172):    at android.os.Handler.dispatchMessage(Handler.java:99)
07-18 11:35:47.337: W/System.err(30172):    at android.os.Looper.loop(Looper.java:137)
07-18 11:35:47.337: W/System.err(30172):    at android.app.ActivityThread.main(ActivityThread.java:5041)
07-18 11:35:47.337: W/System.err(30172):    at java.lang.reflect.Method.invokeNative(Native Method)
07-18 11:35:47.337: W/System.err(30172):    at java.lang.reflect.Method.invoke(Method.java:511)
07-18 11:35:47.337: W/System.err(30172):    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:793)
07-18 11:35:47.337: W/System.err(30172):    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:560)
07-18 11:35:47.337: W/System.err(30172):    at dalvik.system.NativeStart.main(Native Method)
07-18 11:35:47.337: A/chromium(30172): [FATAL:jni_android.cc(319)] Check failed: false. 
07-18 11:35:47.337: A/libc(30172): Fatal signal 11 (SIGSEGV) at 0xdeadbaad (code=1), thread 30172 (l.netmap.client)
```

davisford committed:
new build of Chromium August 6, 2013 for android_webview

davisford committed:
Merge branch 'master' of github.com:davisford/chromeview

davisford committed:
Update vm-build.md

Ubuntu 12.10 fails for Chromium builds

davisford committed:
Update vm-setup.sh

curl is needed for Chromium fonts

schmidp committed:
Some small crbuild fixes

schmidp committed:
make vm-build.sh as executable

schmidp committed:
Under Ubuntu /bin/sh no longer points to bash. Because the build environment depends on running within bash, we use /bin/bash instead.

schmidp committed:
Catching un-initialized variables leads to an
"Unbound variable" error in the "process_options"
function in "build/android/envsetup_functions.sh".

schmidp committed:
Default value for CPUS env. variable

davisford committed:
Merge pull request #1 from openresearch/master

crbuild fixes

jeanpeng commented:
@pwnall  did you had fix this issue,i still has this problem at the master branch

pwnall commented:
@pengjin1024 This is discontinued. The folks at Intel did a much better job than I could've done, so I discontinued this project, and I'm using [Crosswalk](https://crosswalk-project.org/). You should switch to it for a hassle-free WebView.
